### PR TITLE
fix mcp_kWperK list type

### LINKED ISSUE #1217: Photovoltaic thermal is not working in master
This is the error I am getting. @shanshanhsieh: do you know what is happening? The problem is repeated in both mine and Zhongming's systems

C:\Users\Bhargava\Documents\GitHub\CEAforArcGIS\cea\technologies\solar\solar_collector.py:570: RuntimeWarning: overflow encountered in multiply
  1]  # heat production from solar collector, eq.(5)
C:\Users\Bhargava\Documents\GitHub\CEAforArcGIS\cea\technologies\solar\solar_collector.py:570: RuntimeWarning: invalid value encountered in subtract
  1]  # heat production from solar collector, eq.(5)
C:\Users\Bhargava\Documents\GitHub\CEAforArcGIS\cea\technologies\solar\solar_collector.py:584: RuntimeWarning: invalid value encountered in less
  if abs(qdiff < 0.1):
C:/Users/Bhargava/Documents/GitHub/CEAforArcGIS/cea/technologies/solar/photovoltaic_thermal.py:455: RuntimeWarning: invalid value encountered in greater
  if abs(q_balance_error) > 1:
Traceback (most recent call last):
  File "C:\Program Files\JetBrains\PyCharm 2017.3.3\helpers\pydev\pydevd.py", line 1668, in <module>
    main()
  File "C:\Program Files\JetBrains\PyCharm 2017.3.3\helpers\pydev\pydevd.py", line 1662, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "C:\Program Files\JetBrains\PyCharm 2017.3.3\helpers\pydev\pydevd.py", line 1072, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "C:/Users/Bhargava/Documents/GitHub/CEAforArcGIS/cea/technologies/solar/photovoltaic_thermal.py", line 639, in <module>
    main(cea.config.Configuration())
  File "C:/Users/Bhargava/Documents/GitHub/CEAforArcGIS/cea/technologies/solar/photovoltaic_thermal.py", line 617, in main
    latitude=latitude, longitude=longitude, weather_path=config.weather, building_name=building)
  File "C:/Users/Bhargava/Documents/GitHub/CEAforArcGIS/cea/technologies/solar/photovoltaic_thermal.py", line 103, in calc_PVT
    panel_properties_SC, panel_properties_PV, settings)
  File "C:/Users/Bhargava/Documents/GitHub/CEAforArcGIS/cea/technologies/solar/photovoltaic_thermal.py", line 231, in calc_PVT_generation
    absorbed_radiation_PV_Wperm2, T_cell_C, module_area_per_group_m2)
  File "C:/Users/Bhargava/Documents/GitHub/CEAforArcGIS/cea/technologies/solar/photovoltaic_thermal.py", line 531, in calc_PVT_module
    mcp_kWperK[5][x] = 0
TypeError: 'numpy.float64' object does not support item assignment

### PR DESCRIPTION
**Pull request by shanshanhsieh**
this PR resolves #1217 
to test: run `photovoltaic_thermal.py`

### CODE REVIEW

**Comment by shanshanhsieh** [comment]
@JIMENOFONSECA , here is the PR that fixes the bug for `photovoltaic_thermal.py`
the name of the PR was replaced with the description of my commit mysteriously

**Comment by shanshanhsieh** [comment]
branch number `i1217`

File: cea/technologies/solar/photovoltaic_thermal.py [hunk 1]
```diff
@@ -526,16 +524,16 @@ def calc_PVT_module(settings, radiation_Wperm2, panel_properties_SC, panel_prope
             mcp_kWperK = specific_flows_kgpers[flow] * (Cp_fluid_JperkgK / 1000)  # mcp in kW/c
 
     for x in range(8760):
-        if supply_out_total_kW[5][x] <= 0:  # the demand is zero
+        # turn off the water circuit if total energy supply is zero
+        if supply_out_total_kW[5][x] <= 0:
             supply_out_total_kW[5][x] = 0
-            mcp_kWperK[5][x] = 0
+            mcp_kWperK[x] = 0
```
[the remark this answers is not in the document]
Comment (shanshanhsieh) [reply cea/technologies/solar/photovoltaic_thermal.py]: yes, this error is caused by a commit of mine 10 days ago. 
i added `mcp_kWperK` into the for loop without checking the variable type properly, and not sure why it still passed the test on my PC back then...
`mcp_kWperK` has been initiated as a list to store results of the flow=5, unlike the other variables are storing results of all flows.

this is the commit:
https://github.com/architecture-building-systems/CityEnergyAnalyst/commit/adbe5655e2945aebbd1eb452450e78ffe03b5dfc#diff-c86e34ac7d059a65a47ab4be547599de

[the remark this answers is not in the document]
Comment (JIMENOFONSECA) [reply cea/technologies/solar/photovoltaic_thermal.py]: got it thanks

**Pull request merged**